Public/Src/Cache/ContentStore/Distributed/NuCache/EventStreaming/ContentLocationEventStore.cs

```diff
@@ -125,7 +125,21 @@ protected async Task DispatchAsync(OperationContext context, ContentLocationEven
                             context,
                             addContent.Sender,
                             addContent.ContentHashes.SelectList((hash, index) => new ShortHashWithSize(hash, addContent.ContentSizes[index])),
-                            eventData.Reconciling);
+                            eventData.Reconciling,
+                            updateLastAccessTime: true);
+                    }
+
+                    break;
+                case AddContentLocationWithoutTouchingEventData addContentWithoutTouching:

```

Comment:
>case AddContentLocationWithoutTouchingEventData addContentWithoutTouching: [](start = 16, length = 74)

Can we combine this case statement with the case statement above and just have a ternary expression for computing updateLastAccessTime?